ambassador/ambassador/envoy/v2/v2listener.py

```diff
@@ -535,11 +535,15 @@ def __init__(self, config: 'V2Config', listener: IRListener) -> None:
                 } ]
             else:
                 # Use a sane access log spec
+                log_format = config.ir.ambassador_module.get('envoy_log_format', None)
+                if not log_format:
+                    log_format = 'ACCESS [%START_TIME%] \"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\" %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% \"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\" \"%REQ(X-REQUEST-ID)%\" \"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\"'
+                config.ir.logger.info("V2Listener: Using log_format '%s'" % log_format)
```

Comment:
I don't have strong opinions on whether this log line deserves to stay or not. Let me know.

Reply:
I think it should probably be `debug` rather than `info`. The logs are pretty cluttered as is. 🙁